components/backup-stream/src/endpoint.rs

```diff
@@ -66,6 +66,9 @@ const SLOW_EVENT_THRESHOLD: f64 = 120.0;
 /// CHECKPOINT_SAFEPOINT_TTL_IF_ERROR specifies the safe point TTL(24 hour) if task has fatal error.
 const CHECKPOINT_SAFEPOINT_TTL_IF_ERROR: u64 = 24;
 
+/// CHECKPOINT_ADVANCE_USE_V3 use only the v3 checkpoint advance currenly.
+const CHECKPOINT_ADVANCE_USE_V3: bool = true;
```

Comment:
What is `v3`, and what is `v2`?

Reply:
This item is about 2 kinds of method about advancing progress of log-backup.